hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/processors/StreamToStreamJoinP.java

```diff
@@ -80,6 +81,7 @@ public class StreamToStreamJoinP extends AbstractProcessor {
     // package-visible for tests
     final StreamToStreamJoinBuffer[] buffer;
 
+    private final int processorIndex;
```

Comment:
isn't this the same as `localProcessorIndex` (or global?) in `Context` passed to `init`?

Reply:
yes, its the same

Reply:
according to TDD we should use `globalProcessorIndex()`